IlmBase/IlmThread/IlmThread.cpp

```diff
@@ -43,11 +43,11 @@
 
 #include "IlmBaseConfig.h"
 #include "IlmThread.h"
-#include "Iex.h"
+#include "../Iex/Iex.h"

```

Comment:
Did you encounter a bug in include path set up somewhere? We aren't intending that such relative paths should be necessary.

Reply:
I was more comfortable with GCC.

Reply:
We build with gcc as well, but didn't encounter this problem before. Perhaps you could file a new issue, with your build log in it, because the issue of finding Iex is different than solving the mingw thread issues.

Reply:
Ok thanks for the answers. I will just add that there is a bug in IlmThreadMutex.cpp. I changed the record to
# if defined _WIN32 || defined _WIN64 when ILMBASE_FORCE_CXX03
is active.